dub.sdl

```diff
@@ -42,11 +42,110 @@ subPackage {
 
   sourceFiles \
     "src/ddmd/astbase.d" \
-    "src/ddmd/astbasevisitor.d" \
```

Comment:
Same change required as inhttps://github.com/dlang/dmd/pull/7415 (atm moment the DUB file is outdated).